integration-tests/tests/oak_d_test.go

```diff
@@ -64,7 +65,7 @@ func TestCameraServer(t *testing.T) {
 	t.Run("Reconfigure module", func(t *testing.T) {
 		cfg := resource.Config{
 			Attributes: utils.AttributeMap{
-				"sensors": []string{"color"},
+				"sensors": []string{"depth"},
```

Comment:
Oops we could change this back. I just modified it for testing. Pretty arbitrary though